x-pack/test/security_solution_endpoint_api_int/apis/resolver/common.ts

```diff
@@ -547,51 +438,3 @@ export const compareArrays = (
     ).to.be.ok();
   });
 };
-
-/**
- * Verifies that the stats received from ES for a node reflect the categories of events that the generator created.
- *
- * @deprecated use verifyTree
- * @param relatedEvents the related events received for a particular node
- * @param categories the related event info used when generating the resolver tree
- */
-export const verifyEntityTreeStats = (
```

Comment:
Do we still need a check like this?

Reply:
Yeah you can't tell from the diff, but there's one here: https://github.com/elastic/kibana/blob/master/x-pack/test/security_solution_endpoint_api_int/apis/resolver/common.ts#L316

for the new `/tree` api

https://github.com/elastic/kibana/blob/master/x-pack/test/security_solution_endpoint_api_int/apis/resolver/common.ts#L370

The `tree.ts` has a bunch of calls to `verifyTree` which calls `verifyStats`

https://github.com/elastic/kibana/blob/master/x-pack/test/security_solution_endpoint_api_int/apis/resolver/tree.ts#L94